makefile

```diff
@@ -8,12 +13,16 @@ build-backend:
 test-backend:
 	mvn test jacoco:report
 
+test-postman:
+	newman run PurrgentCare.postman_collection.json --env-var "personsBaseURL=$(PERSONS_BASE_URL)" \
+	--env-var "petBaseURL=$(PET_BASE_URL)" --env-var "accountBaseURL=$(ACCOUNT_BASE_URL)"
+
```

Comment:
enabling postman collection to be run from the terminal directly through newman.

You might need to install newman for this to work...
`cd purrgent-care-ui & npm install -g newman`